cmake/config/arm-linux.cmake

```diff
@@ -15,5 +15,6 @@
 set(CMAKE_SYSTEM_NAME Linux)
 set(CMAKE_SYSTEM_PROCESSOR armv7l)
 
-set(CMAKE_C_COMPILER arm-linux-gnueabihf-gcc)
+set(CMAKE_SYSROOT ${CMAKE_EXTERNAL_SYSROOT})
+set(CMAKE_C_COMPILER ${CMAKE_TOOLCHAIN_ROOT}/arm-linux-gnueabihf-gcc)
```

Comment:
Actually if you wanna compile for Kamino, we should use https://github.com/Rokid/ShadowNode/blob/eec21ead1cfdc0857ff1a75cbc388a330c3ff058/cmake/config/openwrt-linux.cmake :)